sound/soc/sof/intel/pci-tgl.c

```diff
@@ -84,6 +84,23 @@ static const struct sof_dev_desc adls_desc = {
 	.ops = &sof_tgl_ops,
 };
 
+static const struct sof_dev_desc adlp_desc = {
+	.machines               = snd_soc_acpi_intel_adl_machines,
+	.alt_machines           = snd_soc_acpi_intel_adl_sdw_machines,
+	.use_acpi_target_states	= true,
```

Comment:
What does `.use_acpi_target_states` for? Do we need `.use_acpi_target_states` fir `adls_desc`, too?

Reply:
@bardliao I talked with Keyon 1 week ago for this question. We need to set this flag. And I had a patch to set this flag on some tgl platforms

Reply:
@bardliao The patch I mentioned is submitted https://github.com/thesofproject/linux/pull/2822. And I described the use_acpi_target_states usage in that PR.